src/data/history.ts

```diff
@@ -83,7 +83,10 @@ export const fetchDate = (
 ): Promise<HassEntity[][]> => {
   return hass.callApi(
     "GET",
-    `history/period/${startTime.toISOString()}?end_time=${endTime.toISOString()}`
+    `history/period/${startTime.toISOString()}
+      ?end_time=${endTime.toISOString()}
+      &significant_changes_only
```

Comment:
You are always adding this to the query string. That's not correct.

Reply:
Why was this resolved? We shouldn't make this change as the backend is backwards compatible.

Reply:
You're right. my fault. I missed this one.